modules/tinymce/src/core/main/ts/focus/EditorFocus.ts

```diff
@@ -14,6 +14,9 @@ import * as FocusController from './FocusController';
 const getContentEditableHost = (editor: Editor, node: Node): HTMLElement | null =>
   editor.dom.getParent(node, (node): node is HTMLElement => editor.dom.getContentEditable(node) === 'true');
 
+const hasContentEditableFalseParent = (editor: Editor, node: Node): boolean =>
+  editor.dom.getParent(node, (node): node is HTMLElement => editor.dom.getContentEditable(node) === 'false') !== null;
```

Comment:
I feel like we have a thousand different ways to check this now, but I can't see an obvious one that implements exactly what you're looking for here.

Reply:
One day we should make a proper library for this, and extract all of this into it.
Or so I like to dream.

Reply:
![xkcd on standards](https://imgs.xkcd.com/comics/standards.png)
https://xkcd.com/927/